plugins/samples/add_header/tests.textpb

```diff
@@ -41,3 +58,38 @@ test {
     result { has_header { key: "Message" value: "foo, bar" } }
   }
 }
+# Tests for header name case insensitivity.
+test {
+  name: "CaseInsensitiveTest"
+  request_headers {
+    # Test framework itself doesn't distinguish case.
```

Comment:
nit: even after reading the code, this comment is confusing to me:

At first, I thought this comment was referring to the fact that the "assertion framework" / "config driven testing framework" ignored the case, but I now see that this is referring to the wasm host interface is what is ignoring the case.

But the wasm plugin itself can't ignore case, since if the wasm plugin enumerates all the headers, it will see case-sensitive strings, right?

But the test framework ignores the case set by the plugin in just the assertions and in get and set operations done by the plugin.

Could you take a whack at restating this comment?

Reply:
Yup, good call. Removed this test and added a clearer comment.